tests/test_mtu.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import pytest
+from netaddr import *
+import time
+import logging
+import requests
+from ptf_runner import ptf_runner
+from ansible_host import AnsibleHost
```

Comment:
This `ansible_host` module will be deprecated. Please stop using it. Use the new fixtures instead: duthost, ptfhost, testbed_devices, etc.

Reply:
@arlakshm, can you address the above comments.

Reply:
addressed in the latest commit